packages/loot-core/migrations/1699454123000_reports.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE reports
+  (id TEXT PRIMARY KEY,
+   name TEXT,
+   start TEXT,
+   end TEXT, 
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏are these dates? If so should be more explicit: `start_date` & `end_date`.

Also something to account for: relative AND absolute timeranges.

Some examples:
- January - December (start of Jan to end of Dec)
- 15th January - 26th January (a specific timerange)
- last 3 months (the specific time range depends on when you're viewing the graph - the data will differ today, tomorrow and a week from now)

I think what you'll want to have is 3x variables: `start_date`, `end_date` and `paused` (boolean). If `paused = true` => that's a specific timerange (1st of Jan to 15th of May). Whereas if `paused = false`, then we can assume this is a sliding window (i.e. last 3 months).